### ISSUE TITLE
clearOnBlur doesn't work with tag-input-dropdown 

### ISSUE THREAD

**Issue by hsuanweifu** [Gbuomprisco/ngx-chips#659]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

On a tag input that's using dropdown the blur action will only hide the dropdown and will not clear the input when clearOnBlur is set to true.

**Expected behavior**

On a tag input that's using dropdown the blur action will clear the input when clearOnBlur is set to true.

**Minimal reproduction of the problem with instructions (if applicable)**

```
    public array = [];
    public testInput = '';
```

```
<tag-input [(ngModel)]='array'
           [(inputText)]="testInput"
           [clearOnBlur]="true">
    <tag-input-dropdown [autocompleteItems]="['one', 'two', 'three']">
    </tag-input-dropdown>
</tag-input>
```

1. Type "o". ("one", "two") shown
2. Click anywhere outside of input the blur the input (dropdown hide text remain)

**What do you use to build your app?. Please specify the version**

Webpack
angular-cli 1.4.7
Windows 10
PHPStorm

**Angular version:** 

4.4.5

**ngx-chips version:**

1.6.3

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chrome

**Comment by jakubbialas** [comment]
There is similar problem with [addOnBlur], when dropdown is opened and I click outside input field (blur), dropdown hide, but value from input is not added.

**Comment by JPMardelli** [comment]
Is there anyway to manually clear the input? Any workarounds?

**Comment by hsuanweifu** [comment]
You have to manually remove the input yourself. There is a text input property you can use to set the input text (see document).

**Comment by Gbuomprisco** [comment]
This should be fixed with v 1.7.2
